### PR TITLE
updating 20 km greenland file to new version

### PR DESCRIPTION
**Pull request by yariseidenbenz**
updating to file name url=‘https://web.lcrc.anl.gov/public/e3sm/inputdata/glc/mpasli/mpas.gis20km/gis_20km_r01.20210824.nc’

### CODE REVIEW

File: compass/landice/tests/greenland/run_model.py [hunk 1]
```diff
@@ -70,7 +70,9 @@ def __init__(self, test_case, velo_solver, name='run_model', subdir=None,
 
         # download and link the mesh
         self.add_input_file(filename='landice_grid.nc',
-                            target='gis20km.210608.nc', database='')
+                            url=‘https://web.lcrc.anl.gov/public/e3sm/
+                                 inputdata/glc/mpasli/mpas.gis20km/
+                                 gis_20km_r01.20210824.nc')
```
Comment (xylar) [inline compass/landice/tests/greenland/run_model.py:75]:
```suggestion
                            url='https://web.lcrc.anl.gov/public/e3sm/
                                 inputdata/glc/mpasli/mpas.gis20km/
                                 gis_20km_r01.20210824.nc')
```
I think maybe a fancy quotation mark got put in at the beginning of the URL by mistake.  It seems to be confusing python.

**Comment by xylar** [comment]
@matthewhoffman and @yariseidenbenz, my preference would be to make a copy (or a symlink) of this file in the [mpas-albany-landice](https://web.lcrc.anl.gov/public/e3sm/mpas_standalonedata/mpas-albany-landice/) standalone database.  Otherwise, every time the test case gets set up, the mesh will have to get downloaded again.  If you add it to the database, it will get cached locally.

**Comment by matthewhoffman** [comment]
@xylar , we were wondering what you would recommend here.  I can imagine going into the future that we will make more use in COMPASS of the exact same meshes we are using in E3SM, and I was worried about having multiple locations with the same file.  Adding a symlink in the `mpas_standalonedata` directory is probably the simplest/cleanest solution.  We'll make the switch to doing it that way.  @yariseidenbenz , I'll let you know when that is ready to try and how to adjust the PR.

**Comment by matthewhoffman** [comment]
@stephenprice , FYI, this is the change for making COMPASS use the new 20km GIS mesh you fixed up for that other PR.

**Comment by matthewhoffman** [comment]
@yariseidenbenz  and @xylar , I pushed a commit with the change Xylar recommended (after creating the symlink in the online Anvil database).  @xylar , feel free to merge or approve (and I can merge).

**Review by xylar — APPROVED** [review]

**Comment by xylar** [comment]
Thanks @yariseidenbenz and @matthewhoffman!

**Pull request merged**